Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -108,20 +76,30 @@ static float AspectToWidescreen(float aspect)
   return aspect * ((16.0f / 9.0f) / (4.0f / 3.0f));
 }
 
-Renderer::Renderer()
+Renderer::Renderer(int backbuffer_width, int backbuffer_height)
 {
+  FramebufferManagerBase::SetLastXfbWidth(MAX_XFB_WIDTH);
+  FramebufferManagerBase::SetLastXfbHeight(MAX_XFB_HEIGHT);
+
   UpdateActiveConfig();
 
+  m_backbuffer_width = backbuffer_width;
+  m_backbuffer_height = backbuffer_height;
+  m_last_efb_scale = g_ActiveConfig.iEFBScale;
+  CalculateTargetSize();
+  UpdateDrawRectangle();
+
   OSDChoice = 0;
   OSDTime = 0;
 }
 
 Renderer::~Renderer()
 {
   // invalidate previous efb format
-  prev_efb_format = PEControl::INVALID_FMT;
+  m_prev_efb_format = PEControl::INVALID_FMT;
```

Comment:
Given these are now non-static, is this and the below line still necessary? If the below code requires it, this should probably be moved into `ShutdownFrameDumping()`